Lib/glyphsLib/builder/layers.py

```diff
@@ -26,6 +25,22 @@ def to_ufo_layer(self, glyph, layer):
         ufo_layer = ufo_font.layers.defaultLayer
     elif layer.name not in ufo_font.layers:
         ufo_layer = ufo_font.newLayer(layer.name)
+    elif layer.name in ufo_font.layers and glyph.name in ufo_font.layers[layer.name]:
+        self.logger.warning(
+            "%s %s: Glyph %s, layer %s: Duplicate glyph layer name"
+            % (
```

Comment:
Shouldn't this be simply `warning("text", arg1, ..., argN)` instead of `%`?

Reply:
yes